reconcile/utils/terrascript/cloudflare_resources.py

```diff
@@ -196,3 +219,32 @@ def populate(self) -> list[Union[Resource, Output]]:
         resources.extend(self._create_cloudflare_certificate_pack(zone, zone_certs))
 
         return resources
+
+
+class CloudflareAccountRolesTerrascriptData:
```

Comment:
Is this used somewhere or was it moved into `CloudflareAccountMemberTerrascriptResource`?

Reply:
removed